index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML/CSS(Microverse-Ali-Raheem</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <nav id="navbar">
+    <div class="container">
+      <div class="navigation-container">
+        <ul class="navigation-list">
+          <li class="navigation-list">
+            <a href="#welcome-section" class="navigation-link">Welcome</a>
+          </li>
+          <li class="navigation-list">
+            <a href="#projects" class="navigation-link">Projects</a>
+          </li>
+          <li class="navigation-list">
+            <a href="#banner-section" class="navigation-link">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <section id="welcome-section">
+    <div class="container">
+        <header id="welcome-header">
+            <div class="text-wrapper">
+                <p class="big">Welcome to my portfilio</p>
+                <h1>I am Microverse</h1>
+                <p class="big">Gain the skills you need to get hired by global companies. Learn online. Connect to life-changing opportunities. Don't pay anything until you land a job!</p>
+            </div>                
+        </header>
+    </div>
+</section>
+<section id="projects">
+    <div class="container">
+        <header id="projects-header">
+            <div class="text-wrapper">
+                <h2>Some of my Projects</h2>
+                <p class="big">HTML | CSS | Java Script</p>
+            </div>                
+        </header>
+        
+        <div class="projects-button">
+            <div class="button-wrapper">
+                <button type="button" >All Projects</button>
+            </div>
+        </div>
+    </div>
+</section>
+<section id="banner-section">
+    <div class="container">
+        <header id="banner-header">
+            <div class="text-wrapper">
+                <p class="big">Thank you for visiting our website</p>
+                <h1>Join Microverse</h1>
+            
+            </div>                
+        </header>
+        <div >
+            <ul class="github" style="list-style-type: none;">
+            
+                <li >
+                    <a href="https://github.com/aliatayee/html-css-Ali-Raheem/" target="_blank" id="profile-link">
+                        <span>vew our GitHub</span>
+                    </a>
+                </li>
+              
+            </ul>
```

Comment:
for the `ul` tag with the class of `github`, please move the inline applied styles to style.css file.
- [ ] please make sure to move the inline styles to style.css file.